src/containers/Cluster/ClusterOverview/components/ClusterMetricsNetwork.tsx

```diff
@@ -0,0 +1,52 @@
+import {DoughnutMetrics} from '../../../../components/DoughnutMetrics/DoughnutMetrics';
+import {formatBytes} from '../../../../utils/bytesParsers';
+import {SHOW_NETWORK_UTILIZATION} from '../../../../utils/constants';
+import {useSetting} from '../../../../utils/hooks/useSetting';
+import i18n from '../../i18n';
+import type {ClusterMetricsBaseProps} from '../shared';
+import {calculateBaseDiagramValues} from '../utils';
+
+import {ClusterMetricsCardContent} from './ClusterMetricsCard';
+
+interface ClusterMetricsNetworkProps extends ClusterMetricsBaseProps {
+    percentsValue: number;
+    throughput?: string;
+}
+
+function formatStorageLegend(value?: string) {
+    return formatBytes({value, withSpeedLabel: true});
+}
+
+export function ClusterMetricsNetwork({
+    percentsValue,
+    throughput,
+    collapsed,
+    ...rest
+}: ClusterMetricsNetworkProps) {
+    const [showNetworkUtilization] = useSetting<boolean>(SHOW_NETWORK_UTILIZATION);
```

Comment:
Why do you need this setting? It is shown only if cluster endpoint supports it (v5)